### PR TITLE
Define equality method for ASN1::ObjectId

### PR DESCRIPTION
**Pull request by mastahyeti**
This makes it easier to compare OIDs.

### CODE REVIEW

File: ext/openssl/ossl_asn1.c [hunk 1]
```diff
@@ -1818,6 +1842,8 @@ do{\
     rb_define_method(cASN1ObjectId, "oid", ossl_asn1obj_get_oid, 0);
     rb_define_alias(cASN1ObjectId, "short_name", "sn");
     rb_define_alias(cASN1ObjectId, "long_name", "ln");
+    rb_define_method(cASN1ObjectId, "==", ossl_asn1obj_eq, 1);
+    rb_define_alias(cASN1ObjectId, "===", "==");
```
Comment (ioquatix) [inline ext/openssl/ossl_asn1.c]: Can we remove this alias, I'm not sure it makes sense.

**Comment by ioquatix** [comment]
@mastahyeti do you mind checking the feedback?

File: ext/openssl/ossl_asn1.c (hunk 1 above)
Comment (mastahyeti) [reply ext/openssl/ossl_asn1.c]: I addressed this in 1351ae3630125eb149d03a15bdc6ae8503c359d2

**Comment by ioquatix** [comment]
LGTM.

**Comment by mastahyeti** [comment]
Thanks. I'm happy to see more activity on this repo!

**Comment by ioquatix** [comment]
Feel free to make PRs I will review them.

**Pull request merged**